configure

```diff
@@ -2274,7 +2274,7 @@ sed < $SRCDIR/arch/generic/Makefile.in "
 # Emscripten does not support large amounts of data via stdin/out
 # https://github.com/emscripten-core/emscripten/issues/16755#issuecomment-1102732849
 if test "$CHOST" != "wasm32"; then
-    TEST="${TEST} ghtests"
```

Comment:
parametrized -> parameterized. I think it i miss spelled everywhere.

Reply:
Fixed.